crypto/crypto_aes_aesni.h

```diff
@@ -23,6 +25,15 @@ void * crypto_aes_key_expand_aesni(const uint8_t *, size_t);
 void crypto_aes_encrypt_block_aesni(const uint8_t[16], uint8_t[16],
     const void *);
 
+/**
```

Comment:
I just remembered that this needs to be inside an `#ifdef CPUSUPPORT_X86_AESNI`.